itest/psbt_test.go

```diff
@@ -1001,17 +1007,21 @@ func testMultiInputPsbtSingleAssetID(t *harnessTest) {
 
 	ConfirmAndAssertOutboundTransfer(
 		t.t, t.lndHarness.Miner.Client, primaryTapd, sendResp,
-		genInfo.AssetId, []uint64{4000, 1000}, 0, 1,
+		genInfo.AssetId, []uint64{changeAmt, sendAmt}, 0, 1,
 	)
 
 	AssertNonInteractiveRecvComplete(t.t, secondaryTapd, 1)
 
-	// Second of two send events from primary (minting) node to the
-	// secondary node.
+	// Second of three send events from primary (minting) node to secondary
+	// node.
+	sendAmt = uint64(1000)
```

Comment:
nit: instead of hard-coding the numbers here maybe calculate them from previous `changeAmt`?

Reply:
I've added a running total for more dynamic change amount calculation.